mmtk/Cargo.toml

```diff
@@ -21,14 +21,15 @@ lazy_static = "1.1"
 # - change branch
 # - change repo name
 # But other changes including adding/removing whitespaces in commented lines may break the CI.
-mmtk = { git = "ssh://git@github.com/mmtk/mmtk-core.git", rev = "b85f621f62c7390029c09e2e136d95e54feea8aa" }
+mmtk = { git = "ssh://git@github.com/mmtk/mmtk-core.git", rev = "0985c60a1d8e4305449731ffc37244f17cc05a44" }
 # Uncomment the following to build locally
 # mmtk = { path = "../repos/mmtk-core" }
 
 [features]
 default = []
 nogc_lock_free = ["mmtk/nogc_lock_free"]
 nogc_no_zeroing = ["mmtk/nogc_no_zeroing"]
+side_gc_header = ["mmtk/side_gc_header"]
```

Comment:
Have we tested that `side_gc_header` works for other plans? Otherwise, the only reason we need this feature is to throw a panic to tell people dont use it for GenCopy while we don't know if it works for other plans.

Reply:
I removed that assertion, so we do not use the feature anymore.
I will remove the feature for now